### DISCUSSION TITLE
Gibt es die ID (DropDown) auch irgendwo im Frontend / Konfiguration?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2161] [Q&A]
Diese ID wird beim Auslösen vom DropDown angezeigt.
![image](https://user-images.githubusercontent.com/84818934/212371003-2d211e52-31b3-4111-96eb-b8c7226db9fd.png)
Kann man diese ID mit dem Gerät verknüpfen, bzw. das Gerät herausfinden?
ID wird bei jedem Neustart vom Frontend neu gesetzt. Keine direkte Identifizierung möglich, nur intern.

**Comment by Zefau** [comment]
Ich vermute die ID ist von dem Framework (Quasar) und nicht von jarvis. Kannst du den Screenshot mal größer machen, damit ich den Kontext sehe?

**Comment by mcuiobroker** [comment]
![image](https://user-images.githubusercontent.com/84818934/212387791-0df31833-aafd-4630-aeda-ac13d69d3bf4.png)

Hab ne Möglichkeit gefunden. Man muss nur alle actionElemente durchsuchen und das "for" -Atrribute auslesen, dann bei Event mit dem vergleichen.
Das event.path

![image](https://user-images.githubusercontent.com/84818934/212389107-5a64fc37-f447-4e05-8edc-05cdbf5bdf47.png)
